solidity/contracts/converter/BancorConverterUpgrader.sol

```diff
@@ -230,9 +244,22 @@ contract BancorConverterUpgrader is IBancorConverterUpgrader, ContractRegistryCl
 
         for (uint16 i = 0; i < connectorTokenCount; i++) {
             address connectorAddress = _oldConverter.connectorTokens(i);
-            IERC20Token connector = IERC20Token(connectorAddress);
-            connectorBalance = connector.balanceOf(_oldConverter);
-            _oldConverter.withdrawTokens(connector, address(_newConverter), connectorBalance);
+            // Ether reserve
+            if (connectorAddress == address(0)) {
+                _oldConverter.withdrawETH(address(_newConverter));
+            }
+            // Ether reserve token

```

Comment:
`Ether reserve token` -> `EtherToken reserve`

Reply:
@yudilevi : We have 66 occurrences of `reserve token` and 0 occurrences of `token reserve` in our contract documentation. Are you sure?

Reply:
As above - not critical